src/pages/settings/Wallet/WalletEmptyState.js

```diff
@@ -34,7 +35,7 @@ function WalletEmptyState({onAddPaymentMethod}) {
     const {translate} = useLocalize();
     return (
         <IllustratedHeaderPageLayout
-            backgroundColor={themeColors.walletPageBG}
+            backgroundColor={themeColors.PAGE_BACKGROUND_COLORS[SCREENS.SETTINGS.WALLET]}
             illustration={LottieAnimations.FastMoney}
```

Comment:
note no need to QA this as users will no longer see this screen since no user will have an empty wallet. We will remove this page in a follow up after the conference